Exec/science/nova/problem_initialize.H

```diff
@@ -27,13 +27,23 @@ void problem_initialize ()
         }
     }
 
-    // velocity perturbation stuff
+    // velocity perturbation x-axis vortex location
 
     Real offset = (probhi[0] - problo[0]) / problem::num_vortices;
 
     for (int i = 0; i < problem::num_vortices; i++) {
         problem::xloc_vortices[i] = (static_cast<Real>(i) + 0.5_rt) * offset + problo[0];
     }
+
+    problem::ih1 = network_spec_index("hydrogen-1");
+    problem::layer_h1_base = 0.0_rt;
+
+    for (int n = 0; n < model::npts; n++) {
+        if (model::profile(0).state(n, model::ispec+problem::ih1) > 0.5_rt) {
+            problem::layer_h1_base = model::profile(0).r(n);
+            break;
```

Comment:
you never declare `problem::layer_h1_base` -- that should be added to the `_prob_params` file

Reply:
yes, I forgot to add it  _prob_params! In the run is added, otherwise, it will not work.